modules/examples/eb-sf-batch/images/simulation-mock/src/simulation_mock/__init__.py

```diff
@@ -0,0 +1,35 @@
+#  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Can we delete this if we're not really using it in the module?

Reply:
Deleted. Thanks for identifying